src/Native/Bootstrap/CppCodeGen.h

```diff
@@ -41,4 +41,33 @@ inline double __uint64_to_double(uint64_t v)
     return val.d;
 }
 
-#endif // __CPP_CODE_GEN_H
+#ifdef USE_PORTABLE_HELPERS
+struct PInvokeTransitionFrame
+{
+#ifdef _TARGET_ARM_
+    TgtPTR_Void     m_ChainPointer; // R11, used by OS to walk stack quickly
+#endif
+    TgtPTR_Void     m_RIP;
```

Comment:
What is the minimal `struct PInvokeTransitionFrame` that we can get away for `USE_PORTABLE_HELPERS` ? Ideally, it would have just `m_pThread` and nothing else.